ui/components/multichain/app-header/app-header.js

```diff
@@ -135,9 +135,16 @@ export const AppHeader = ({ location }) => {
       >
         <>
           <Box
-            className={classnames('multichain-app-header__contents', {
-              'multichain-app-header-shadow': isUnlocked && !popupStatus,
-            })}
+            className={classnames(
+              `${
+                isUnlocked
+                  ? 'multichain-app-header__contents'
+                  : 'multichain-app-header__lock-contents'
+              }`,
```

Comment:
We should not need template string here, no?

```suggestion
              isUnlocked
                ? 'multichain-app-header__contents'
                : 'multichain-app-header__lock-contents',
```